understanding/22/target-size-minimum.html

```diff
@@ -42,50 +42,47 @@ <h4>Target Size (Minimum)</h4>
+            <li>If the spacing of the targets is essential to the information being conveyed, the "Essential" exception applies. For example, in digital maps, the position of pins is analogous to the position of places shown on the map. If there are many pins close together, the offset between pins and neighboring pins will often be below 24 CSS pixels. It is essential to show the pins at the correct map location, therefore the Essential exception applies.</li>
         </ol>
 
         <p>The requirement is independent of the zoom factor of the page; when users zoom in the CSS pixel size of elements does not change. This means that authors cannot meet it by claiming that the target will have enough spacing or sufficient size if the user zooms into the page.</p>
 
-        <p>While the Success Criterion primarily helps touch users by ensuring that activation with the 'fat finger' will not accidentally trigger adjacent targets, it is also useful for mouse or pen users. It reduces the chances of erroneous activation due to a tremor or some other input imprecision.</p>
+        <p>While the Success Criterion primarily helps touch users in not accidentally triggering adjacent targets, it is also useful for mouse or pen users. It reduces the chances of erroneous activation due to either a tremor or reduced precision, whether because of reduced fine motor control or input imprecision.</p>
       
-        <p>As mobile devices generally have internal heuristics to identify which link is closest, spacing can work as effectively as a larger target size.</p>
+        <h3>Spacing versus size</h3>
+        <p>As mobile devices generally have internal heuristics to identify which link is closest, spacing can work as effectively as a larger target size. The following illustrate a variety of situations. In the top line of icons, the dimensions of each icon target are 44 by 44 CSS pixels with no space in between. This amply passes this Success Criterion and is actually sufficient to meet the AAA requirement for <a href="../21/target-size-enhanced.html">Target Size (Enhanced)</a>. In the next row, the dimensions of each icon target are 24 by 24 CSS pixels with no space in between, passing this Success Criterion. In the third row, the the same icon targets are only 20 by 20 CSS pixels but they have a 4 CSS pixel space in between, passing this Success Criterion via the spacing exception. In the last row, the the same icon targets are only 20 by 20 CSS pixels and have no space in between, failing this Success Criterion.</p>
```

Comment:
"As mobile devices generally have internal heuristics" seems to imply here that the SC is only really aimed at mobile devices, which it isn't. touch on desktop/laptop devices, mouse, stylus, etc are all covered, and this "heuristics on mobile" part doesn't address those scenarios.

Reply:
Mulling this over (as it's the last comment that wasn't addressed), perhaps moving that first sentence from where it is now (so starting the section directly with "The following...") to just before the "Spacing versus size" heading, and morphing it to:

```Note: touchscreen devices/user agents generally have internal heuristics to identify which link or control is closest to a user's touch interaction. In these cases, providing sufficient spacing around a target can work as effectively as a larger target size itself.```

would help actually embed this thought more logically into the flow, so it doesn't come out of nowhere AND doesn't seem to just focus the whole section purely on mobile.

Reply:
Got back from holiday, I can't tell whether @patrickhlauke's comments were added before/after the version approved in the meeting, do you know @mbgower?

Reply:
after. i do think they're quite important though to remove the weird non-sequitur of talking straight about mobile heuristics, would be good to get a second (third?) approval, otherwise i'll file a new PR with that once merged.

Reply:
to be clear, the unresolved one here was after the meeting (as that was the last outstanding one that hadn't been discussed in the meeting, at least going by the minutes)